hunting/aws/docs/multiple_service_logging_deleted_or_stopped.md

```diff
@@ -0,0 +1,42 @@
+# Multiple Service Logging Deleted or Stopped
+
+---
+
+## Metadata
+
+- **Author:** Elastic
+- **Description:** This hunting query identifies the deletion or stopping of multiple service logging actions within AWS. Service logging is a critical security control that provides visibility into the activities and changes within AWS services. Adversaries may attempt to disable or delete service logging to evade detection and cover their tracks. Monitoring for multiple service logging deletions or stops can help identify potential malicious activity and ensure that critical security controls remain intact.
+
+- **UUID:** `d74f8928-5e46-11ef-9488-f661ea17fbce`
+- **Integration:** [aws.cloudtrail](https://docs.elastic.co/integrations/aws/cloudtrail)
+- **Language:** `[ES|QL]`
+- **Source File:** [Multiple Service Logging Deleted or Stopped](../queries/multiple_service_logging_deleted_or_stopped.toml)
+
+## Query
+
+```sql
+from logs-aws.cloudtrail*
+| where @timestamp > now() - 7 day
+| where
+    event.provider in ("ec2.amazonaws.com","route53resolver.amazonaws.com","s3.amazonaws.com", "cloudtrail.amazonaws.com")
+    and event.action in ("DeleteFlowLogs","DeleteResolverQueryLogConfig", "DeleteTrail", "StopLogging")
+| eval date = DATE_FORMAT("YYYY-mm-dd", @timestamp)
```

Comment:
to express the `Multiple` aspect do we need to add a stats count > threshold here ?

Reply:
I left this generic on purpose as any of this could be blinding to monitoring or visibility for threats so even 1 occurrence of "StopLogging" should be verified and checked.

To your point, I did aggregate by provider (AWS service), event action, event outcome, date and ARN. Even if outcome is failed, it would be a good indication of attempts to do so with valid account compromise but maybe not the correct permissions.